privacy/bolton/README.md

```diff
@@ -0,0 +1,56 @@
+# Bolton Subpackage
+
+This package contains source code for the Bolton method. This method is a subset
+of methods used in the ensuring privacy in machine learning that leverages
```

Comment:
I'm having trouble understanding this sentence. What is the larger set that the "subset" is referring to? This sentence is also vague enough that I think perhaps it should be combined with the next section, which is more specific and clearer.